File: BackEnd/api/model/lastCheck_model.py
import peewee
from datetime import datetime
from api.model.host_model import Host

from api.utils.db import db

class LastCheck(peewee.Model):
    host = peewee.ForeignKeyField(Host, backref='lastCheck')
    ping = peewee.CharField()
    cpuUsage = peewee.CharField()
    cpuName = peewee.CharField()
    ramUsed = peewee.CharField()
    ramFree = peewee.CharField()
    ramCached = peewee.CharField()
    netIn = peewee.CharField()
    netOut = peewee.CharField()

    class Meta:
        database = db

File: FrontEnd/src/app/models/lastcheck.model.ts
import { HostModel } from "./host.model"

export interface LastCheckModel{
    
    hostName: string,
    hostIp: string,
    ping: string,
    cpuUsage: number,
    cpuName: string,
    ramUsed: number,
    ramFree: number,
    ramCached: number,
    netIn: number,
    netOut:number
    
}